app-ng/src/entry-point.lisp

```diff
@@ -0,0 +1,226 @@
+;;;; app-ng/src/entry-point.lisp
+;;;;
+;;;; Author: appleby
```

Comment:
It's probably polite to keep old authors where code has been copied.

Reply:
This occurred to me as well, but only after making my comment above about wanting to drop it entirely :)

Reply:
Dropping the filename + authors section? I would be very much in favour

Reply:
Oops, I forgot that comment thread was in the other PR that got closed.

I am right there with you, but @stylewarning's got the veto :)

Reply:
FWIW, I'd also favor dropping that header. After all, we can quickly see who's responsible for a piece of code using `git blame`

Reply:
This is done. I've copied over the old "Authors".